app/models/edition.rb

```diff
@@ -30,6 +30,9 @@ class Edition < ActiveRecord::Base
   has_many :classification_featurings, inverse_of: :edition
   has_many :links_reports, as: :link_reportable
 
+  has_many :dependencies, class_name: 'EditionDependency', foreign_key: :edition_id, dependent: :destroy
```

Comment:
Hi Tekin: your former suggestion is not unworkable, it does work. i've tried that in the beginning. only limiting point is that the scope doesn't help create associated objects, like a `has_many` would, so i replaced that with the `has_many` on `Edition`. i've taken your suggestions for the association names, good one!